src/third-party/PhishingController.test.ts

```diff
@@ -69,186 +69,123 @@ describe('PhishingController', () => {
+      sinon.useFakeTimers();
+      const controller = new PhishingController({ refreshInterval: 10 });
 
-    const controller = new PhishingController({});
-    await controller.test('metamask.io');
+      expect(controller.isOutOfDate()).toBe(false);
+    });
 
-    expect(nockScope.isDone()).toBe(true);
-  });
+    it('should not be out of date after some of the refresh interval has passed', () => {
+      const clock = sinon.useFakeTimers();
+      const controller = new PhishingController({ refreshInterval: 10 });
+      clock.tick(5);
 
-  it('does not call fetch twice if the second call is inside of the refreshInterval', async () => {
-    const nockScope = nock(PHISHING_CONFIG_BASE_URL)
-      .get(METAMASK_CONFIG_FILE)
-      .reply(200, {
-        blacklist: [],
-        fuzzylist: [],
-        tolerance: 0,
-        whitelist: [],
-        version: 0,
-      })
-      .get(PHISHFORT_HOTLIST_FILE)
-      .reply(200, []);
+      expect(controller.isOutOfDate()).toBe(false);
+    });
 
-    const controller = new PhishingController({});
-    await controller.test('metamask.io');
+    it('should be out of date after the refresh interval has passed', () => {
+      const clock = sinon.useFakeTimers();
+      const controller = new PhishingController({ refreshInterval: 10 });
+      clock.tick(10);
 
-    expect(nockScope.isDone()).toBe(true);
-    // The nock handlers were not created with `persist()`, so they will only
-    // be used once. The next `fetch` will throw, so this expectation shows
-    // that no `fetch` has occurred.
-    expect(await controller.test('metamask.io')).toStrictEqual({
-      result: false,
-      type: 'all',
+      expect(controller.isOutOfDate()).toBe(true);
     });
-  });
 
-  it('should call fetch twice if the second call is outside the refreshInterval', async () => {
-    const clock = sinon.useFakeTimers(Date.now());
-    const nockScope = nock(PHISHING_CONFIG_BASE_URL)
-      .get(METAMASK_CONFIG_FILE)
-      .times(2)
-      .reply(200, {
-        blacklist: [],
-        fuzzylist: [],
-        tolerance: 0,
-        whitelist: [],
-        version: 0,
-      })
-      .get(PHISHFORT_HOTLIST_FILE)
-      .times(2)
-      .reply(200, []);
+    it('should be out of date if the refresh interval has passed and an update is in progress', async () => {
+      const clock = sinon.useFakeTimers();
+      const controller = new PhishingController({ refreshInterval: 10 });
+      clock.tick(10);
+      const pendingUpdate = controller.updatePhishingLists();
 
-    const controller = new PhishingController();
-    await controller.test('metamask.io');
-    await clock.tickAsync(defaultRefreshInterval);
-    await controller.test('metamask.io');
+      expect(controller.isOutOfDate()).toBe(true);
+
+      // Cleanup pending operations
+      await pendingUpdate;
+    });
 
-    expect(nockScope.isDone()).toBe(true);
+    it('should not be out of date if the configuration was just updated', async () => {
```

Comment:
total nit, but it will be out of date if you update the configuration to `0`

Reply:
Sorry I'm not sure what you mean here. Do you mean if `refreshInterval` was zero?

If `refresInterval` is zero, the config will always be outdated.

Reply:
Yeah... That's what I meant. The phrasing `should not be out of date if the configuration was just updated` implies that it should not be out of date if the configuration was updated **with any value**. It really doesn't matter though.

Reply:
Oh, I see, that's not what I meant by "configuration". I meant the phishing configuration, i.e. the JSON config being fetched

Reply:
I can reword these to use the term "phishing lists" to avoid confusing it with the controller config

Reply:
Done here: https://github.com/MetaMask/controllers/pull/929/commits/21d9b47c59eceb6fcd4554115c69d077fa9fd0e8